packages/contentful-migrations/ui/1100000000001-create-m-stage-migration.js

```diff
@@ -146,12 +155,23 @@ module.exports = async function (migration, context) {
       linkType: 'Entry',
     });
 
-  mStage.changeFieldControl('name', 'builtin', 'singleLine', {});
-  mStage.changeFieldControl('title', 'builtin', 'singleLine', {});
-  mStage.changeFieldControl('subtitle', 'builtin', 'singleLine', {});
+  mStage.changeFieldControl('internal_name', 'builtin', 'singleLine', {
+    helpText: 'e.g. "Home page > Stage"',
+  });
+
+  mStage.changeFieldControl('overline', 'builtin', 'singleLine', {});
+
+  mStage.changeFieldControl('headline', 'builtin', 'singleLine', {});
+
+  mStage.changeFieldControl('subline', 'builtin', 'singleLine', {});
+
   mStage.changeFieldControl('text', 'builtin', 'richTextEditor', {});
+
   mStage.changeFieldControl('media', 'builtin', 'entryLinkEditor', {});
+
   mStage.changeFieldControl('links', 'builtin', 'entryLinksEditor', {});
+
   mStage.changeFieldControl('layout', 'builtin', 'dropdown', {});
+
   mStage.changeFieldControl('content', 'builtin', 'entryLinksEditor', {});
```

Comment:
Can't add a comment to the validations of `content` so i'll add it here.
We may need some sort of `c-text` component ;)

Reply:
See #31